composer.json

```diff
@@ -94,7 +94,7 @@
 	},
 	"config": {
 		"platform": {
-			"php": "7.0"
+			"php": "7.2"
```

Comment:
@MrPetovan  - This line does the trick, see https://getcomposer.org/doc/06-config.md#platform
I can now assume using PHP 7.2 for installing dependencies, but the minimum requirements are still PHP 7.0.

Therefore I can require PHP-Unit with `--ignore-platform-reqs` into the `composer.json` and pin it at `composer.lock`, but there's still no failure during install. The only thing is that when installing the whole dependencies, the unit-tests are now just usable for PHP 7.2+

Reply:
This is not good either. Since Friendica *can* run on PHP 7.0, the dependencies should also work. This means setting the platform value to the same as the minimum PHP requirement value so that we pull the correct dependencies.

With your change, we could pull a PHP 7.2+ dependency version which would break Friendica for PHP 7.0-7.1 even though Friendica's own code could work on PHP 7.0.

The dev dependencies do not need to be bound to this version though, so using `--ignore-platform-reqs` is warranted to update PHPUnit.

Reply:
I'm finally seeing your comment about backward-breaking changes in PHPUnit which I didn't think about. In this case it is better to use the PHPUnit version compatible with PHP 7.2 and lock that version in the `composer.lock` file for use with all the test environments. You can temporarily change the PHP version in `platform` to do this.

Reply:
OK found another way => I'm manually installing the PHPUnit 8 phar per CI and so I don't need to adapt the composer files  and routines at all :-)